src/components/Header.tsx

```diff
@@ -3,11 +3,15 @@
 import {Link, Navbar, NavbarBrand, NavbarContent, NavbarItem} from '@nextui-org/react';
 import React from 'react';
 import {Button} from '@nextui-org/button';
-import {useRouter} from 'next/navigation';
+import {usePathname, useRouter} from 'next/navigation';
+import SearchBar from '@/components/SearchBar';
 
 export default function Header() {
   const router = useRouter();
 
+  const pathname = usePathname() || '';
+  const titlepageRegex = /^\/\d+$/;
```

Comment:
Er vi sikker på at vi bare vil ha den på titteloversikt-sidene? Jeg ser ikke et problem med å ha den på redigering. Kunne vært alle unntatt startsiden for min del

Reply:
Jeg har ingen store preferanse. Koden blir litt lettere med det du foreslår (`pathname === '/'` er nok, vi slipper RegExp).

Men jeg ser ikke for meg at å hoppe fra redigering til en annen tittel er en vanlig scenario.

Reply:
Sikkert ikke, merger slik det er så finner vi eventuelt ut etter hvert!